pycryptobot.py

```diff
@@ -168,7 +168,7 @@ def getAction(now: datetime = datetime.today().strftime('%Y-%m-%d %H:%M:%S'), ap
         action = 'WAIT'
 
     # if disabled, do not buy within 3% of the dataframe close high
-    if app.disableBuyNearHigh() is True and (price > (df['close'].max() * 0.97)):
```

Comment:
I think this is correct but, please pay extra attention this part.

I think this could cause stopping sell operation.
Also was giving false logs when last action is 'BUY'.

Reply:
Ok, thanks.